src/pages/ServiceDetails/ServiceDetailsPage.tsx

```diff
@@ -272,7 +282,6 @@ class ServiceDetails extends React.Component<ServiceDetailsProps, ServiceDetails
                 <div>Inbound Metrics</div>
               </NavItem>
               {errorTraces !== undefined &&
-                errorTraces > -1 &&
```

Comment:
This is reverting of change https://github.com/kiali/kiali-ui/pull/1188/files
Will it keep the logic "when Traces is not configured it should hide Traces tab"?

Reply:
Yes because we are checking down  if there is a url this.props.jaegerUrl